podcasts/HeadphoneSettingsViewController.swift

```diff
@@ -189,10 +189,7 @@ class HeadphoneSettingsViewController: PCTableViewController {
                 switch self {
                 case .bookmarkSound:
                     // Only show this option if the user has selected addBookmark as one of the options
-                    let optionEnabled = Settings.isPlayBookmarkCreationSoundAvailable
-
-                    // and the FeatureFlag is enabled
-                    return FeatureFlag.bookmarks.enabled && optionEnabled
+                    return  Settings.isPlayBookmarkCreationSoundAvailable
```

Comment:
There's a double space here

```suggestion
                    return Settings.isPlayBookmarkCreationSoundAvailable
```